exercises/practice/strain/.meta/Example.vb

```diff
@@ -0,0 +1,24 @@
+Imports System
+Imports System.Collections.Generic
+Imports System.Runtime.CompilerServices
+
+Public Module Strain
+    <Extension()>
+    Public Function KeepMethod(Of T)(ByVal collection As IEnumerable(Of T), ByVal predicate As Func(Of T, Boolean)) As IEnumerable(Of T)
+        Return ApplyPredicate(collection, predicate)
```

Comment:
```suggestion
    Public Function Keep(Of T)(ByVal coll As IEnumerable(Of T), ByVal pred As Func(Of T, Boolean)) As IEnumerable(Of T)
        Return ApplyPredicate(coll, pred)
```

Collection and Predicate are case-insensitive keywords, so best not to use them for variable names.